src/update_variety_data.py

```diff
@@ -35,16 +35,27 @@
 
 def _create_variety_data(config):
     full_variety_path = os.path.join(get_src_dir(), config['data_storage']['variety_path'])
```

Comment:
if you migrated `os` -> `Path`, I would have merged it immidiatly :wink:

Reply:
changed to Path